homeassistant/components/lovelace/__init__.py

```diff
@@ -143,6 +143,10 @@
             return
 
         if change_type == collection.CHANGE_ADDED:
+
+            if "-" not in url_path:
```

Comment:
I don't think that we should do this. Instead we should enforce it during validation. Existing ones from beta we can have a upgrade-on-load code.